DOCS/interface-changes.rst

```diff
@@ -79,7 +79,6 @@ Interface changes
     - remove the `--tone-mapping-crosstalk` option
     - add `--gamut-mapping-mode=perceptual|relative|saturation|absolute|linear`
     - add `--corner-rounding` option
-    - change `--subs-with-matching-audio` default from `yes` to `no`
```

Comment:
That did change with 0.36.0 and now that it's released, that list should be immutable imo.

That would be a change for 0.36.1 (or w/e the next version is)

Reply:
I put it under 0.37.0